composables/useIdentity.ts

```diff
@@ -27,7 +27,7 @@ export default function useIdentity({
   )
 
   const { data, refetch, loading } = useGraphql({
-    clientName: computed(() => (isDotAddress.value ? 'pid' : 'kid')),
+    clientName: computed(() => (isDotAddress.value ? 'pid' : 'kid')).value,
```

Comment:
Why? The `clientName` could be a computed value so that it would be reactive.

Reply:
I've already done that in all queries so we'll need to change many lines of code

Reply:
It should be a reactive value, or it would introduce some bugs. I could expand more about how it causes bugs if needed.

I see there is a new implementation on `useGraphql`. However, it should keep the original logic that allows `clientName` to be reactive. 
<img width="1107" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/c33f0bbf-7b15-4143-92db-db7b47e0fce6">

Reply:
idk why but https://github.com/kodadot/nft-gallery/blob/5ed65d90db3af697bf7e900ede2d0b9e29d9bbfa/composables/useGraphql.ts#L12 doesn't work as excepted I had to pass the client value on each gql call otherwise it didn't work (not reactive, it would keep old prefix)
If you can push a proof of work where `useGaphql` doesn't need to pass `clientName` I would be glad to see it

Reply:
> @roiLeo Could you pls provide a small case that which function or page could produce this bug? I'm going to solve it but I cannot produce it now.

Check out `useCarouselSpotlight` query, I had snek result that point on bsx elements

Reply:
I see. I will raise a PR to fix the root issue.

Reply:
> idk why but
> 
> https://github.com/kodadot/nft-gallery/blob/5ed65d90db3af697bf7e900ede2d0b9e29d9bbfa/composables/useGraphql.ts#L12
> 
> doesn't work as excepted I had to pass the client value on each gql call otherwise it didn't work (not reactive, it would keep old prefix)
> If you can push a proof of work where `useGaphql` doesn't need to pass `clientName` I would be glad to see it

> Check out useCarouselSpotlight query, I had snek result that point on bsx elements

Here is the proof of work that we do not need to pass `clientName` for each ggl call. #7306


> > doesn't work as excepted I had to pass the client value on each gql call otherwise it didn't work (not reactive, it would keep old prefix)

As far as I know, the old prefix only occurs on the local dev env because router does not redirect properly. I also find that there are lots of warnings related to the router. Idk how to fix it. If you check any of the preview env of nuxt branch, you will find that it works well.